{"name": "test_load_yaml", "status": "broken", "statusDetails": {"message": "UnicodeDecodeError: 'gbk' codec can't decode byte 0xac in position 23: illegal multibyte sequence", "trace": "data_dir = WindowsPath('D:/api-pythonproject/data')\n\n    def test_load_yaml(data_dir):\n        file_path = data_dir / 'goods_cat.yaml'\n        with open(file_path)as f:\n>           data =yaml.safe_load(f)\n\ntest_yaml.py:7: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\venv\\lib\\site-packages\\yaml\\__init__.py:125: in safe_load\n    return load(stream, SafeLoader)\n..\\venv\\lib\\site-packages\\yaml\\__init__.py:79: in load\n    loader = Loader(stream)\n..\\venv\\lib\\site-packages\\yaml\\loader.py:34: in __init__\n    Reader.__init__(self, stream)\n..\\venv\\lib\\site-packages\\yaml\\reader.py:85: in __init__\n    self.determine_encoding()\n..\\venv\\lib\\site-packages\\yaml\\reader.py:124: in determine_encoding\n    self.update_raw()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <yaml.loader.SafeLoader object at 0x03B82AD0>, size = 4096\n\n    def update_raw(self, size=4096):\n>       data = self.stream.read(size)\nE       UnicodeDecodeError: 'gbk' codec can't decode byte 0xac in position 23: illegal multibyte sequence\n\n..\\venv\\lib\\site-packages\\yaml\\reader.py:178: UnicodeDecodeError"}, "start": 1679037721190, "stop": 1679037721191, "uuid": "5de22126-abe4-4e7a-bc2a-576a4fe1c6d7", "historyId": "25447fd47bf8e902eebd91c8bd9ffec9", "testCaseId": "6caa69dc7d8db51ef22b3ec5f3db517e", "fullName": "testcases.test_yaml#test_load_yaml", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_yaml"}, {"name": "host", "value": "DESKTOP-A7I9MKJ"}, {"name": "thread", "value": "6892-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_yaml"}]}